opencti-platform/opencti-graphql/graphql-codegen.yml

```diff
@@ -12,6 +12,8 @@ generates:
     config:
       useTypeImports: true
       useIndexSignature: true
+      maybeValue: T | null | undefined
+      inputMaybeValue: T | null
```

Comment:
what is it used for ?

Reply:
Because I want to be able to send "undefined" as an answer for a graphql endpoint : the notifierTest one. No message = everything went well. A message = Error message.